src/blight/cli.py

```diff
@@ -26,14 +27,35 @@ def _export_guess_wrapped():
         _export(f"BLIGHT_WRAPPED_{variable}", tool_path)
 
 
+def _swizzle_path():
+    blight_dir = f"/tmp/blight-XXX"
+    if not os.path.isdir(blight_dir):
+        os.mkdir(blight_dir)
+
+    for variable, tool in blight.tool.TOOL_ENV_MAP.items():
+        tool_path = shutil.which(tool)
```

Comment:
I think you can get rid of this check: it's okay for these tools to not be on the `$PATH`, since they could be passed as absolute paths to the build system.